pkg/controller/config/config.go

```diff
@@ -91,9 +91,9 @@ var (
 		},
 		NodeConfig: NodeConfig{
 			DefaultDeadlines: DefaultDeadlines{
-				DefaultNodeExecutionDeadline:  config.Duration{Duration: time.Hour * 48},
-				DefaultNodeActiveDeadline:     config.Duration{Duration: time.Hour * 48},
-				DefaultWorkflowActiveDeadline: config.Duration{Duration: time.Hour * 72},
+				DefaultNodeExecutionDeadline:  config.Duration{Duration: time.Hour * 0},
+				DefaultNodeActiveDeadline:     config.Duration{Duration: time.Hour * 0},
+				DefaultWorkflowActiveDeadline: config.Duration{Duration: time.Hour * 0},
 			},
```

Comment:
I just did some minor testing locally - if you remove the DefaultDeadlines default configuration entirely then these values will be automatically set to '0'. I think that makes more sense than explicitly setting these values.

Reply:
>(2) The [flyte repo](https://github.com/flyteorg/flyte) can compile a single binary with all of the Flyte components. This is the easiest way to test changes. You can use `go replace github.com/flyteorg/flytepropeller => <path to local flytepropeller> to compile a single binary with your local changes to the flytepropeller repository. This can then be run with something like: `./flyte start --config ./flyte_local.yaml`The Flyte UI will be available on`localhost:30080/console` and can be used to execute workflows using the local k3d cluster.


I'll try this today, this seems easier